var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -40,32 +40,54 @@ class VtkM(CMakePackage, CudaPackage):
     variant('build_type', default='Release', description='CMake build type',
             values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
     variant("shared", default=False, description="build shared libs")
-    variant("cuda", default=False, description="build cuda support")
+
     variant("doubleprecision", default=True,
             description='enable double precision')
     variant("logging", default=False, description="build logging support")
     variant("mpi", default=False, description="build mpi support")
-    variant("openmp", default=(sys.platform != 'darwin'), description="build openmp support")
     variant("rendering", default=True, description="build rendering support")
-    variant("tbb", default=(sys.platform == 'darwin'), description="build TBB support")
     variant("64bitids", default=False,
             description="enable 64 bits ids")
 
+    # Device variants
+    variant("cuda", default=False, description="build cuda support")
+    variant("openmp", default=(sys.platform != 'darwin'), description="build openmp support")
+    variant("tbb", default=(sys.platform == 'darwin'), description="build TBB support")
```

Comment:
Do these two conflict or is it possible to build both/neither?

Reply:
It is possible to build both/neither . Currently we support any number of CPU backends, but only 1 GPU backend.